### PR TITLE
Fix all issues pertaining to UG and Edit Command

### PR DESCRIPTION
**Pull request by Blizzeracz**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ Adds an animal to the address book.
 Format: `add n/NAME i/ID g/SEX s/SPECIES b/BREED db/DOB da/DOA`
 
 * ID must be a unique 4 digit number.
+* SEX must be either `male` or `female`.
```
Comment (bkjwjason) [inline docs/UserGuide.md:73]: I think it would be good to specify whether case-sensitive or not here!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -70,6 +70,9 @@ Adds an animal to the address book.
...
+* DOB and DOA must be in the format `YYYYMMDD`.
+* Species and breed can only be alphabetic with only a single space between words.
```
Comment (bkjwjason) [inline docs/UserGuide.md:75]: For this, might be good to specify how many words can it contain! I think our implementation only allows 2 words max with a space between, but this line might imply unlimited number of words as long as there are spaces

**Review by bkjwjason — CHANGES_REQUESTED** [review]
Good job! Just have a look at my comments and let me know if it makes sense

File: docs/UserGuide.md (hunk 1 above)
Comment (Blizzeracz) [reply docs/UserGuide.md:73]: Yep. That sounds better!

File: docs/UserGuide.md (hunk 2 above)
Comment (Blizzeracz) [reply docs/UserGuide.md:75]: Yep you are right!

**Review by bkjwjason — APPROVED** [review]
LGTM!

**Pull request merged**